DNN Platform/Components/ClientDependency/Source/DependencySorter.cs

```diff
@@ -10,30 +10,6 @@ namespace ClientDependency.Core
     /// </summary>
     internal static class DependencySorter
     {
-
-        public static IList<IClientDependencyFile> FilterDependencies(IList<IClientDependencyFile> dependencies)
```

Comment:
I believe this is an addition from @donker which allows framework deduplication

Reply:
Code restored.